### ISSUE TITLE
Hi! Are there video tutorials?

### ISSUE THREAD

**Issue by dolearndouble** [jakzo/Il2CppDecompiler#1]
Hi! Are there video tutorials?

**Comment by jakzo** [comment]
No, just the instructions in the readme. I haven't put much effort into it since I got LLM decompilation working because GPT4 avoids decompiling large functions at all cost which makes it really annoying. If I ever get back to this I'll try improving the algorithmic approach I started with.

**Comment by dolearndouble** [comment]
Will the tools in your repository actually work? Can they successfully convert to C#?

**Comment by jakzo** [comment]
They work. Not guaranteed correct because LLM can make mistakes but most functions I tried decompiled correctly.

**Comment by dolearndouble** [comment]
hi，how to use it work in an android apk?
